### ISSUE TITLE
Size of LOCATION window

### ISSUE THREAD

**Issue by terence1016** [Ar1i/PokemonGo-Bot#3932]
I have the problem of inputing longitude and latitude in the LOCATION Window below the map, just like the following picture. This problem was discovered when GET POINT function added. Im using Windows 7 (64bit).

![111](https://cloud.githubusercontent.com/assets/22253200/18606852/0efe36be-7cee-11e6-9526-2270b69ea65e.jpg)

**Comment by Schnitzelvernichter** [comment]
3939
you have to delete the user settings in the Graphic

in the operating system of the computer

**Comment by depeco** [comment]
Sorry i really don't understand your answer !
Someone can  explain us the way to do that ?

**Comment by depeco** [comment]
Up..
Can't bot for 5 days i don't understand how to resolve this problem.
Someone can explain me please.

**Comment by Xelwon** [comment]
try changing font dpi to 92 and put your windows teme to the default of windows 7.
I have that configuration and works perfect.
